group.lua

```diff
@@ -128,20 +127,37 @@ function AnimationGroup:GetLoopState()
     return self.loop_state
 end
 
+function AnimationGroup:GetAnimations()
+    local animations = {}
+    for i = 0, AG.ORDER_LIMIT - 1, 1 do
+        for _, animation in next, self.animations[i + 1] or {} do
```

Comment:
I think it's more readable to have a condition before the loop:

```suggestion
        if self.animations[i + 1] then
            for _, animation in next, self.animations[i + 1] do
```

Reply:
reworked